lib/v2/txrjy/templates/fornumtopic.art

```diff
@@ -0,0 +1,6 @@
+<h4>{{ author }}</h4>
```

Comment:
Do not put author in description

Reply:
The author is used to mark who commented on this post,
So I think it would be better to keep it.

Reply:
:ok_hand: saw it